source/iface/vector-types.rst

```diff
@@ -8,6 +8,34 @@
 Vector types
 ************
 
+SYCL provides a cross-platform class template that works
+efficiently on SYCL devices as well as in host C++ code.
+This type allows sharing of vectors between the host and its
+SYCL devices. The vector supports member functions that allow
+construction of a new vector from a swizzled set of component
+elements.
+
+``vec<typename DataT``, int NumElements> is a vector type that compiles
+down to a SYCL backend built-in vector types on SYCL devices, where
+possible, and provides compatible support on the host or when it is
```

Comment:
```suggestion
possible, and provides compatible support on the host when 
```

Reply:
I've opened [this ](https://github.com/KhronosGroup/SYCL-Docs/pull/522) around these two proposed changes

Reply:
As I understand agreed removal of full first sentence in the second paragraph: 
_``vec<typename DataT, int NumElements>`` is a vector type that compiles
down to a SYCL backend built-in vector type on SYCL devices, where
possible, and provides compatible support on the host or when it is
not possible._ 
Done with commit : https://github.com/KhronosGroup/SYCL_Reference/pull/140/commits/10a0e11a4a1cfa16ba887d881ad24f7a806151b6